src/gfx/drawable_processors/mesh_drawable_processor.hpp

```diff
@@ -380,14 +299,20 @@ struct MeshDrawableProcessor final : public IDrawableProcessor {
       for (size_t index = 0; index < context.drawables.size(); ++index) {
         const IDrawable *drawable = context.drawables[index];
         auto &drawableData = drawableDatas->emplace_back();
-        generateDrawableData(drawableData, context.context, cachedPipeline, drawable, context.viewData, context.frameCounter,
-                             needProjectedDepth);
+
+        ParameterStorage *baseDrawData{};
+        if (context.generatorData.drawParameters) {
+          baseDrawData = &(*context.generatorData.drawParameters)[index];
+        }
+
+        // TODO: Temporary to store per-view textures in per-object bind group
```

Comment:
still todo?

Reply:
Yes, although it only has a slight performance/memory impact since per-view texture bindings are duplicated across object currently.
I plan to address it later. See => #580